docs/pages/kubernetes-access/helm/reference/teleport-cluster.mdx

```diff
@@ -741,6 +795,62 @@ Sets the `Group` of `Issuer` to be used when issuing certificates with `cert-man
   </TabItem>
 </Tabs>
 
+### `highAvailability.certManager.issuerSpec`
+
+| Type | Default value | Can be used in `custom` mode? | `teleport.yaml` equivalent |
+| - | - | - | - |
+| `object` | `{}` | ❌ | None |
+
+Spec of `Issuer` or `ClusterIssuer`.
+
+<Admonition type="note" title="Installing cert-manager">
+  You must install and configure `cert-manager` in your Kubernetes cluster yourself.
+
+  See the [cert-manager Helm install instructions](https://cert-manager.io/docs/installation/kubernetes/#option-2-install-crds-as-part-of-the-helm-release)
+  and the relevant sections of the [AWS](../guides/aws.mdx#step-47-install-and-configure-cert-manager) and [GCP](../guides/gcp.mdx#step-47-install-and-configure-cert-manager) guides for more information.
+
+  You must create this yourself when `highAvailability.certManager.createIssuer` is `true`.
```

Comment:
See comments above about removing `createIssuer`.

Reply:
I fixed it by 290e52289394097dcc2d30cd3ec8357528e491c6 .